index.html

```diff
@@ -364,6 +365,7 @@ <h2 class="contact">Contact me</h2>
           <input type="submit" value="Get It Touch" />
```

Comment:
- [ ] The submit button should be aligned to the left instead of the center as seen in the Figma design
### Your implementation
![image](https://user-images.githubusercontent.com/45460192/165120757-ff58d8c7-ca49-47f0-801c-5af67d5f5391.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/165120852-afe533c0-6c82-4af2-9aa6-3c9591ecd27c.png)
- [ ] Please implement the pseudo-classes as required by the Figma to this submit button. It should look like these
![image](https://user-images.githubusercontent.com/45460192/165121307-aeb42dd9-77cd-4dce-89c4-49b0c5747b1f.png)